openlibrary/plugins/openlibrary/code.py

```diff
@@ -658,7 +658,7 @@ def is_admin(self):
         return u and u.is_admin()
 
     def GET(self, key):
-        # only allow admin users to edit yaml
+        # only allow admin users to edit yml
```

Comment:
```suggestion
        # only allow admin users to edit yaml
```
I apologize for not being as clear as I should have been. For things that were _already_ `yaml`, we should leave them that way. :)